test/integration/simple_example/controls/gcloud.rb

```diff
@@ -0,0 +1,28 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+
+title 'Cloud DNS Terraform GCP Test Suite'
+
+gcp_project_id = attribute('gcp_project_id')
+
+control 'cloud-dns-zone' do
```

Comment:
If inspec-gcp doesn't have all of resources we need to write comprehensive tests, let's fall back to shelling out to `gcloud` and write all our tests there.